projects/knora-ui/src/lib/viewer/values/uri-value/uri-value.component.html

```diff
@@ -1,19 +1,25 @@
-<span [formGroup]="form">
-    <mat-form-field class="large-field child-value-component" floatLabel="never">
-        <input matInput [formControlName]="'uriValue'" class="value" placeholder="Uri value" type="url" [readonly]="mode === 'read'">
-    </mat-form-field>
-    <mat-form-field class="large-field value-component-comment" *ngIf="shouldShowComment">
-        <textarea matInput
-                  cdkTextareaAutosize
-                  #autosize="cdkTextareaAutosize"
-                  cdkAutosizeMinRows="1"
-                  cdkAutosizeMaxRows="6"
-                  [formControlName]="'comment'"
-                  class="comment"
-                  placeholder="Comment"
-                  type="text"
-                  [readonly]="mode === 'read'"
-                  spellcheck="false">
-          </textarea>
-      </mat-form-field>
-</span>
+<div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
+    <p class="rm-value">{{valueFormControl.value}}</p>
+    <p class="rm-comment" *ngIf="shouldShowComment">{{commentFormControl.value}}</p>
+</div>
+<ng-template #showForm>
+    <span [formGroup]="form">
+        <mat-form-field class="large-field child-value-component" floatLabel="never">
+            <input matInput [formControlName]="'uriValue'" class="value" placeholder="Uri value" type="url" [readonly]="mode === 'read'">
```

Comment:
readonly

Reply:
Will be done in https://dasch.myjetbrains.com/youtrack/issue/DSP-206